Don't automatically use Homebrew paths.
MacPorts already sets the correct paths to use.
--- CMakeLists.txt.orig	2019-11-19 10:55:14.000000000 -0600
+++ CMakeLists.txt	2020-07-07 12:40:45.000000000 -0500
@@ -69,39 +69,6 @@
     SET(CMAKE_INSTALL_PREFIX ${CMAKE_BINARY_DIR}/dist CACHE PATH "" FORCE)
 ENDIF()
 
-# Find default search paths for OS X; adapted from Stack Overflow question
-# 1487752.
-IF (APPLE)
-    EXECUTE_PROCESS(COMMAND which port
-        RESULT_VARIABLE DETECT_MACPORTS
-        OUTPUT_VARIABLE MACPORTS_PREFIX
-        ERROR_QUIET OUTPUT_STRIP_TRAILING_WHITESPACE)
-    EXECUTE_PROCESS(COMMAND brew --prefix
-        RESULT_VARIABLE DETECT_HOMEBREW
-        OUTPUT_VARIABLE HOMEBREW_PREFIX
-        ERROR_QUIET OUTPUT_STRIP_TRAILING_WHITESPACE)
-
-    IF (${DETECT_MACPORTS} EQUAL 0)
-        GET_FILENAME_COMPONENT(MACPORTS_PREFIX ${MACPORTS_PREFIX} DIRECTORY)
-        GET_FILENAME_COMPONENT(MACPORTS_PREFIX ${MACPORTS_PREFIX} DIRECTORY)
-        SET(CMAKE_LIBRARY_PATH ${MACPORTS_PREFIX}/lib ${CMAKE_LIBRARY_PATH})
-        SET(CMAKE_INCLUDE_PATH ${MACPORTS_PREFIX}/include ${CMAKE_INCLUDE_PATH})
-        MESSAGE(STATUS "Detected MacPorts installation: ${MACPORTS_PREFIX}")
-    ELSE()
-        UNSET(MACPORTS_PREFIX)
-    ENDIF()
-
-    IF (${DETECT_HOMEBREW} EQUAL 0)
-        SET(CMAKE_LIBRARY_PATH ${HOMEBREW_PREFIX}/lib ${CMAKE_LIBRARY_PATH})
-        SET(CMAKE_INCLUDE_PATH ${HOMEBREW_PREFIX}/include ${CMAKE_INCLUDE_PATH})
-        MESSAGE(STATUS "Detected Homebrew installation: ${HOMEBREW_PREFIX}")
-    ELSE()
-        UNSET(HOMEBREW_PREFIX)
-    ENDIF()
-
-    UNSET(DETECT_HOMEBREW)
-    UNSET(DETECT_MACPORTS)
-ENDIF()
 
 # Attempt to retrieve git branch and SHA1 hash of current changeset.
 INCLUDE(GetGitRevisionDescription)
